pnp-helper/src/main/java/samples/com/microsoft/azure/sdk/iot/helper/PnpHelper.java

```diff
@@ -0,0 +1,73 @@
+package samples.com.microsoft.azure.sdk.iot.helper;
```

Comment:
I have plaved the helper under the package `com.microsoft.azure.sdk.iot.pnphelpers`: https://github.com/Azure/azure-iot-sdk-java/blob/abmisr/thermostat/pnp/src/main/java/com/microsoft/azure/sdk/iot/pnphelpers/PnpHelper.java

What do you think of this as the folder/ package structure?

I would rather not have the word "samples" in the package and folder path.

Reply:
> Don't forget to include the log4j.properties file for this new sample so that users who run it can easily get logs from the SDK. The other other samples each have a resources folder that you can copy to do that

Let me take a look at that. Thanks. Will reach out if I have any questions.

Reply:
> I have plaved the helper under the package `com.microsoft.azure.sdk.iot.pnphelpers`: https://github.com/Azure/azure-iot-sdk-java/blob/abmisr/thermostat/pnp/src/main/java/com/microsoft/azure/sdk/iot/pnphelpers/PnpHelper.java
> 
> What do you think of this as the folder/ package structure?
> 
> I would rather not have the word "samples" in the package and folder path.

I think it is a good idea to get rid of the samples as we discussed that this might be released separately. And we can use the pnp-helper for the module name. I believe your change will go in first so make those changes and I can rebase. I'll just add my helper functions to that.